<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph</class>
 <widget class="QDialog" name="graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>701</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_x_min">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_x_max">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_min">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X MIN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_max">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X MAX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_build_graph">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>15</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Build Graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input_expr">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>15</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
